test/build.gradle

```diff
@@ -4,7 +4,6 @@ apply plugin: 'kotlin-kapt'
 
 android {
     compileSdkVersion COMPILE_SDK_VERSION
-    buildToolsVersion BUILD_TOOLS_VERSION
```

Comment:
Don't we need it?

Reply:
Setting `buildToolsVersion` has been optional since 3.0.0. I removed it so we don't have to update it manually anymore.
I can revert this if you prefer to set `buildToolsVersion` explicitly.

Reply:
OK! I'm fine for the change.